fees/buffer/index.ts

```diff
@@ -0,0 +1,50 @@
+import { CHAIN } from "../../helpers/chains";
+import { request, gql } from "graphql-request";
+import type { ChainEndpoints } from "../../adapters/types"
+import { Chain } from '@defillama/sdk/build/general';
+import BigNumber from "bignumber.js";
+import { getTimestampAtStartOfDayUTC } from "../../utils/date";
+import { Adapter } from "../../adapters/types"
+
+const endpoints = {
+  [CHAIN.ARBITRUM]: "https://api.thegraph.com/subgraphs/name/bufferfinance/buffer-mainnet"
+}
+
+
+const graphs = (graphUrls: ChainEndpoints) => {
+  return (chain: Chain) => {
+    return async (timestamp: number) => {
+      const dateId = Math.floor(getTimestampAtStartOfDayUTC(timestamp) / 86400)
+
+      const graphQuery = gql
+      `{
+        dailyRevenueAndFee(id: ${dateId}) {
+          totalFee
+          settlementFee
+        }
+      }`;
+
+      const graphRes = await request(graphUrls[chain], graphQuery);
+      const dailyRev = new BigNumber(graphRes.dailyRevenueAndFee.settlementFee).div(1000000);
+      const dailyFee = new BigNumber(graphRes.dailyRevenueAndFee.totalFee).div(1000000);
+
+      return {
+        timestamp,
+        dailyFees: dailyFee.toString(),
+        dailyRevenue: dailyRev.toString(),
+      };
+    };
+  };
+};
+
+
+const adapter: Adapter = {
+  adapter: {
+    [CHAIN.ARBITRUM]: {
+        fetch: graphs(endpoints)(CHAIN.ARBITRUM),
+        start: async ()  => 56585555 ,
```

Comment:
could you change start `56585555` to unix timestamp
if use this start time is `Sun Oct 17 1971 22:12:35 GMT+0000`

Reply:
This is done

Reply:
Let us know if anything else